README.md

```diff
@@ -32,56 +32,71 @@
 </p>
 
 ## 📦 Install Dependencies
+
 Running flyteconsole locally requires [NodeJS](https://nodejs.org) and
 [yarn](https://yarnpkg.com). Once these are installed, you can run application locally.
 For help with installing dependencies look into
 [Installation section](CONTRIBUTING.md#-install-dependencies).
 
 ## 🚀 Quick Start
+
 1. Follow [Start a Local flyte backend](https://docs.flyte.org/en/latest/getting_started/index.html), like:
     ```bash
     docker run --rm --privileged -p 30080:30080 -p 30081:30081 -p 30082:30082 -p 30084:30084 cr.flyte.org/flyteorg/flyte-sandbox
-    ``` 
+    ```
 2. Now, export the following env variables:
 
-    ``
-    export ADMIN_API_URL=http://localhost:30080
-    export DISABLE_AUTH=1
-    ``
+    `export ADMIN_API_URL=http://localhost:30080 export DISABLE_AUTH=1`
 
-   > You can persist these environment variables either in the current shell or in a `.env` file at the root
-     of the repository. A `.env` file will persist the settings across multiple terminal
-     sessions.
+    > You can persist these environment variables either in the current shell or in a `.env` file at the root
+    > of the repository. A `.env` file will persist the settings across multiple terminal
+    > sessions.
 
 3. Start the server (uses localhost:3000)
 
-    ``bash
-    yarn start
-    ``
+    `bash yarn start `
```

Comment:
remove bash from here please.
 I know this is not you change,, but formating changed it a bit wrong.

Reply:
Fixed 👍